### PR TITLE
feat(frontend): enhance dapp data

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

The Design-Team added more social links and new images for the dapp-explorer. This MR updates adds the new data.

# Changes

- Add images for decideID and OpenChat
- New image for KongSwap
- Add callToAction as optional string
- Add OpenChat and Telegram as social

### CODE REVIEW

File: src/frontend/src/lib/types/dappDescription.ts [hunk 1]
```diff
@@ -14,13 +14,16 @@ const dAppDescriptionSchema = z.object({
 	description: z.string(),
 	stats: z.string(),
 	logo: z.string(),
+	callToAction: z.string().optional(),
```
Comment (peterpeterparker) [inline src/frontend/src/lib/types/dappDescription.ts:17]: given that `DappDescription` matches the types of the portal and II, can you rather add the new specific fields for Oisy as extension?

For example:

interface DappDescription {}
interface OisyDappDescription extends DappDescription {}

(just used a dummy name to explain what I meant)

**Review by peterpeterparker — COMMENTED** [review]
Could you confirm if the new PNG files are optimized for the best resolution and compression? For example, woould similar files in Webp significantly smaller?

File: src/frontend/src/lib/types/dappDescription.ts (hunk 1 above)
Comment (loki344) [reply src/frontend/src/lib/types/dappDescription.ts:17]: Yes I like this idea.

**Comment by loki344** [comment]
> Could you confirm if the new PNG files are optimized for the best resolution and compression? For example, woould similar files in Webp significantly smaller?

Converting to webp and compressing once more saved a few KB :)

File: src/frontend/src/lib/types/oisyDappDescription.ts [hunk 2]
```diff
@@ -27,14 +27,19 @@ const dAppDescriptionSchema = z.object({
 	video: z.string().optional(),
 	videoContentType: z.enum(['video/webm', 'video/mp4']).optional(),
 
-	submittableId: z.string().optional(),
+	submittableId: z.string().optional()
+});
 
-	featured: z.boolean().optional()
+const oisyDappDescriptionSchema = dAppDescriptionSchema.extend({
+	featured: z.boolean().optional(),
```
Comment (peterpeterparker) [inline src/frontend/src/lib/types/oisyDappDescription.ts:34]: not for this PR but, I guess this can become `true` actually?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/types/oisyDappDescription.ts (hunk 2 above)
Comment (loki344) [reply src/frontend/src/lib/types/oisyDappDescription.ts:34]: Sorry ,I don't understand your question.

Comment (peterpeterparker) [reply src/frontend/src/lib/types/oisyDappDescription.ts:34]: Nvm my comment, my bad.

**Pull request merged**